File: airbyte-integrations/connectors/source-oracle-v2/src/main/java/io/airbyte/integrations/source/oracle/OracleSource.java
package io.airbyte.integrations.source.oracle;

import io.airbyte.cdk.core.AirbyteConnectorRunner;
import io.airbyte.cdk.core.IntegrationCommand;

public class OracleSource {

  static public void main(String[] args) {
    AirbyteConnectorRunner.run(AirbyteConnectorRunner.ConnectorType.SOURCE, IntegrationCommand.class, args);
  }
}

File: airbyte-integrations/connectors/source-oracle-v2/src/main/kotlin/io/airbyte/integrations/source/oracle/OracleSource.kt
package io.airbyte.integrations.source.oracle

import io.airbyte.cdk.core.IntegrationCommand
import io.airbyte.cdk.core.context.AirbyteConnectorRunner

fun main(args: Array<String>) {
    AirbyteConnectorRunner.run(IntegrationCommand::class.java, *args)
}
